src/main/java/frc/robot/subsystems/Conveyor.java

```diff
@@ -17,6 +19,15 @@
             FRONT_RATE = 0.01,
```

Comment:
rates are unnnessary, you arent useing them anywhere

Reply:
The rates are in use for defining the rate limiters.